src/assets/icons/icon-loading.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="20px" height="20px" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>icon-loading</title>
+    <defs>
+        <radialGradient cx="50%" cy="29.7778058%" fx="50%" fy="29.7778058%" r="58.2576821%" gradientTransform="translate(0.500000,0.297778),scale(1.000000,0.969469),rotate(44.960928),scale(1.000000,2.384566),translate(-0.500000,-0.297778)" id="radialGradient-1">
+            <stop stop-color="#FFFFFF" offset="0%"/>
+            <stop stop-color="#FFFFFF" stop-opacity="0" offset="100%"/>
+            <stop stop-color="#FFFFFF" stop-opacity="0" offset="100%"/>
+        </radialGradient>
+    </defs>
+    <g id="icon-loading" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <path d="M10,0 C12.9163855,0 15.5411836,1.24843419 17.3691234,3.24003169 L15.928922,4.62888771 C14.4652288,3.01416721 12.3510501,2 10,2 C5.581722,2 2,5.581722 2,10 C2,14.418278 5.581722,18 10,18 C13.4577175,18 16.4030835,15.8063628 17.5203341,12.7348524 L19.3893874,13.4487372 C17.9845471,17.272311 14.3108165,20 10,20 C4.4771525,20 0,15.5228475 0,10 C0,4.4771525 4.4771525,0 10,0 Z" id="Combined-Shape" fill="url(#radialGradient-1)"/>
+    </g>
+</svg>
```

Comment:
We should probably run this through an svg minimiser.

Reply:
minimized using https://vecta.io/nano:
https://github.com/sparkletown/sparkle/pull/1426/commits/880f966fdbeb06065f5946456902e13410b88582